components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -19,7 +19,7 @@ info:
     This document specifies a **RESTful API** for WSO2 **API Manager** - **Publisher**.
 
     # Authentication
-    The Publisher REST API is protected using OAuth2 and access control is achieved through scopes. Before you start invoking
+    The Publisher REST APIs are protected using OAuth2 and access control is achieved through scopes. Before you start invoking
```

Comment:
One publisher API with multiple operations. @tgtshanika please confirm.

Reply:
This is fixed now.